examples/webpack/src/app.js

```diff
@@ -3,6 +3,7 @@ import '@compiled/react';
 import { Suspense, lazy } from 'react';
 
 import { primary } from './common/constants';
+import { CustomFileExtension } from './ui/custom-file-extension.customjsx';
```

Comment:
The custom file extension should be for an imported value. Importing a component like this won't be transformed by compiled. Same for the parcel example.